### DISCUSSION TITLE
Creating a solution with name from the Editor !

### DISCUSSION THREAD

**Discussion by MyelinsheathXD** [ezEngine/ezEngine#520] [Ideas]
Any plans to create a solution via editor with a specific name?

**Comment by jankrassnigg** [comment]
The solution will have the same name as the project. Currently I don't see a need for setting a different name.
This way the name is predictable and thus also easier for the rest of the code to know what the solution file name is and whether one already exists.

**Reply by MyelinsheathXD** [reply]
Oh thanks!  I didn't noticed this simple logic before. XD
